dali/server/daserver.cpp

```diff
@@ -752,6 +766,7 @@ int main(int argc, const char* argv[])
         if (ok) {
             writeSentinelFile(sentinelFile);
             covenMain();
+            initializeMetrics(serverConfig);
```

Comment:
this isn't going to work here, covenMain enters a loop and will not return until Dali starts to quit.
Should be moved before covenMain call

Reply:
@jakesmith Can it simply be moved up one line?

Reply:
yes that is enough afaics.